content/docs/recipes/upgrade-kafka.md

```diff
@@ -0,0 +1,289 @@
+
+# Upgrade KUDO Kafka 
+
+This runbook explains how to upgrade a running KUDO Kafka to a newer version of KUDO Kafka. 
+
+  * [Preconditions](#Preconditions)
+  * [Steps](#steps)
+    + [Verifying if KUDO Kafka is ready for the upgrade](#verifying-if-kudo-kafka-is-ready-for-the-upgrade)
+      - [1. Get the KUDO Kafka Instance object name](#1-get-the-kudo-kafka-object-name)
+      - [2. Verify the KUDO Kafka instance plans are COMPLETE](#2-verify-the-kudo-kafka-plans-are-complete)
+      - [3. Get the operatorversion  of KUDO Kafka Instance](#3-get-the-operatorversion--of-kudo-kafka)
+      - [4. Get the installed operator versions of KUDO Kafka](#4-get-the-installed-operator-versions-of-kudo-kafka)
+    + [Preparing for the upgrade](#preparing-for-the-upgrade)
+      - [5. Verify the inter.broker.protocol.version is matching the current OperatorVersion](#5-verify-the-interbrokerprotocolversion-is-matching-the-current-operatorversion)
+      - [6. Update the inter.broker.protocol.version to match the current version](#6-update-the-interbrokerprotocolversion-to-match-the-current-version)
+      - [7. Install the new OperatorVersion using --skip-instance](#7-install-the-new-operatorversion-using---skip-instance)
+      - [8. Verify the new OperatorVersion is installed correctly](#8-verify-the-new-operatorversion-is-installed-correctly)
+    + [Upgrade the KUDO Kafka Instance](#upgrade-the-kudo-kafka)
+      - [9. Upgrade the KUDO Kafka Instance](#9-upgrade-the-kudo-kafka)
+      - [10. Verify the Kafka version through the pod logs](#10-verify-the-kafka-version-through-the-pod-logs)
+      - [11. Verify the Kafka version through the container version](#11-verify-the-kafka-version-through-the-container-version)
+      - [12. Verify the Kafka version through the app version of the installed KUDO Instance](#12-verify-the-kafka-version-through-the-app-version-of-the-installed-kudo-instance)
+      - [13. (Optional) Bump the `inter.broker.protocol.version` to upgraded version](#13--optional--bump-the--interbrokerprotocolversion--to-upgraded-version)
+  * [Future improvements](#future-improvements)
```

Comment:
This seems to verbose for a runbook.   the description should not be a TOC IMO.     There also doesn't seem like a need to "jump" to a specific part of a runbook.  I would expect to need to run through the entire runbook.